scripts/generate_test_data.py

```diff
@@ -371,3 +437,57 @@ def generate_hpo_files(genes):
             if hgnc_symbol in genes:
                 outfile.write(line+'\n')
         LOG.info("File ready")
+
+def read_panel_file(lines):
+    """Read a file with gene ids and names.
+    
+    A file with genes. First column hgnc id, secon column hgnc symbol
+    """
+    genes = {}
+    for line in lines:
+        if line.startswith('#'):
+            continue
+        line = line.split('\t')
+        if len(line) < 3:
+            continue
+        hgnc_id = int(line[0])
+        hgnc_symbol = line[1]
+        genes[hgnc_symbol] = hgnc_id
+    
+    return genes
+
+@click.group()
+@click.pass_context
```

Comment:
I'm wondering if this will be affected by the changes I made in the main CLI (the appcontext). Where does this function gets its context from?

Reply:
No this is just within the same script. Its the old click context that we used before